lldb/source/Core/Declaration.cpp

```diff
@@ -71,7 +71,7 @@ int Declaration::Compare(const Declaration &a, const Declaration &b) {
 }
 
 bool Declaration::FileAndLineEqual(const Declaration &declaration) const {
-  int file_compare = FileSpec::Compare(this->m_file, declaration.m_file, true);
+  int file_compare = FileSpec::Compare(this->m_file, declaration.m_file, false);
```

Comment:
Why was this changed?

Reply:
Better to do it on the up-and-up.  I added a full parameter and set it in the two places this function is used.